Yoast/wordpress-seo
# Split start to prestart and start

## Disclaimer
Tiny tiny issue with not much of an actual use-case. Sorry Dieter!
One plus is semantic; the separation of concerns: the build and the dev server start.

## Summary

This PR can be summarized in the following changelog entry:

* [non-user-facing] Splits the start script into prestart and start. Props @atimmer 

## Relevant technical choices:

* When in the sitation of having to restart the dev server a lot. This PR will let you only compile it once, with a quick deletion of the prestart line. This works by taking advantage of the `pre` hook: see https://yarnpkg.com/lang/en/docs/package-json/#toc-scripts

## Test instructions

This PR can be tested by following these steps:

* Install packages: `composer install && yarn`.
* Run the dev server: `yarn start`.
* Look at the output of the start command:
  * Ensure the `prestart` commands are run; which should be something like this:
<img width="394" alt="prestart" src="https://user-images.githubusercontent.com/35524806/51258343-0aedc480-19aa-11e9-80f4-c38c309d3b61.png">
  * Ensure `start` command is run; which should be something like this:
<img width="889" alt="start" src="https://user-images.githubusercontent.com/35524806/51258395-28229300-19aa-11e9-98f9-317b24c188ef.png">


### To test my use-case
* Stop the dev server by pressing `ctrl+c`
* Remove the `prestart` line in `package.json`.
* Run the dev server again: `yarn start`.
* Ensure only the start command is run.

## Quality assurance


Fixes #
